src/python/pants/option/global_options.py

```diff
@@ -152,6 +158,9 @@ def from_bootstrap_options(cls, bootstrap_options):
     process_execution_local_enable_nailgun=False,
     remote_cache_read=False,
     remote_cache_write=False,
+    remote_store_initial_timeout=10,
+    remote_store_timeout_multiplier=1.0,
```

Comment:
I think that defaulting to exponential backoff is probably reasonable... so 2.0 or something.

Reply:
I wanted to keep the existing timeout to not change behavior, but I agree, 2.0 should be reasonable.